README.md

```diff
@@ -204,6 +204,17 @@ invoke it again for incremental data pulls using the query_data function.
 task definitions and prepares the payload as per the entity shape of the destination connector and hands it over to the
 destination connector by invoking the `writeData` function.
 
+## SDK Restrictions
+- For `queryData` and `writeData` calls, AppFlow expects the connectors to respond within 30 seconds.
+  If the Lambda is taking more than 30 seconds, then that will result in a timeout and connector error will be surfaced to the customer.
+  For use cases where it might take more than 30 seconds, please use the pagination in such a way where each page can be fetched under 30 seconds .
+- For `describeConnectorConfiguration`, `validateConnectorRuntimeSettings`, `validateCredentials`, `describeEntity` and `listEntities` calls,
+  AppFlow expects the connectors to respond within 10 seconds. If the Lambda is taking more than 10 seconds then that will result in a timeout and connector error will be surfaced to the customer.
```

Comment:
If the Lambda invocation takes more than 30 seconds then that will result in a timeout and connector error will be surfaced to the customer.